api/infrastructure/parallelcluster-api.yaml

```diff
@@ -964,6 +964,17 @@ Resources:
       ManagedPolicyArns:
         - !Sub arn:${AWS::Partition}:iam::aws:policy/AmazonSSMManagedInstanceCore
         - !Sub arn:${AWS::Partition}:iam::aws:policy/EC2InstanceProfileForImageBuilderECRContainerBuilds
+      Policies:
+        - PolicyName: ImageBuilderCustomPolicy
+          PolicyDocument:
+            Version: '2012-10-17'
+            Statement:
+              - Effect: Allow
+                Action: ecs:RegisterContainerInstance
+                Resource: !Sub arn:${AWS::Partition}:ecs:*:${AWS::AccountId}:cluster/*
```

Comment:
Is it possible to scope the permission down by restricting more the resource pattern, e.g. including the StackIdSuffix?

Reply:
Let's go with this implementation for this iteration.
By the way, let's be sure to ask ECS suggestions on how to scope it down.
For instance, what about considering (**not in this PR**) to scope it down using conditions on tags?